twine-cert.txt

```diff
@@ -2407,11 +2407,11 @@
     de = Prüfregeln
     en = Checking rules
     
-[settings_rules_list_status_available]
-    de = Aktuell
+[settings_rules_list_status_updated]
+    de = Aktualisiert
     en = Up to date
```

Comment:
"Up to date" is not the correct translation of "Aktualisiert".

Reply:
true that - good catch 😉 I guess "Up to date" would not be a neckbreaker but let me fix it 👍

Reply:
Thanks @alexcimander! Looks good now!